jax/lax/lax_control_flow.py

```diff
@@ -1440,7 +1436,7 @@ def scan_bind(*args, reverse, length, num_consts, num_carry, jaxpr, linear):
     assert len(linear) == len(args)
     consts, init, xs = split_list(args, [num_consts, num_carry])
     consts_avals, init_avals, x_avals = split_list(jaxpr.in_avals, [num_consts, num_carry])
-    xs_avals = _map(partial(_promote_aval_rank, length), x_avals)
+    # xs_avals = _map(partial(_promote_aval_rank, length), x_avals)
```

Comment:
Delete?

Reply:
I left it commented because the variable is used in a commented-out check a few lines below. If we delete this line, we should probably delete the other one as well.

Reply:
Done.